test/integration/targets/ansible-galaxy-role/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+- file:
+    state: directory
+    path: "{{item}}"
+  loop:
+  - /tmp/testrole.d/meta
```

Comment:
I recommend looking at how other tests create a temporary dir per invocation to avoid collisions/hardcoding

Reply:
Done